bigquery_etl/metadata/parse_metadata.py

```diff
@@ -146,7 +207,26 @@ def write(self, file):
                 if label_value == "":
                     metadata_dict["labels"][label_key] = True
 
-        file.write_text(yaml.dump(metadata_dict))
+        # Use literal yaml representation for descriptions
+        class Literal(str):
+            pass
+
+        def literal_presenter(dumper, data):
+            return dumper.represent_scalar("tag:yaml.org,2002:str", data, style="|")
+
+        yaml.add_representer(Literal, literal_presenter)
+
+        if "description" in metadata_dict:
+            metadata_dict["description"] = Literal(metadata_dict["description"])
+
+        converter = cattr.Converter()
+        file.write_text(
+            yaml.dump(
+                converter.unstructure(metadata_dict),
+                default_flow_style=False,
+                sort_keys=False,
```

Comment:
This ensures that keys appear in the `metadata.yaml` file in the order they are defined in the `Metadata` class. Which means the friendly name and description are at the top of the file

Reply:
This is great!